src/components/EarningsTable/EarningsTable.styles.js

```diff
@@ -0,0 +1,185 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+/**
+ * @constant
+ * @type {object}
+ * @description possible colors for earning table component
+ * @see {@link https://colors.artyclick.com/color-name-finder/}  - to learn how
+ *  color names are generated
+ */
+const COLORS = {
```

Comment:
Any colors we use should ideally be part of the palette in `src/components/common/theme.js` so that we can maintain consistency across tools.
`primary` is the Greenstand green we're using, based on the latest style guidelines.

Reply:
I will move the colors there